twins/api/http/transport.go

```diff
@@ -280,7 +280,7 @@ func readMetadataQuery(r *http.Request, key string) (map[string]interface{}, err
 	m := make(map[string]interface{})
 	err := json.Unmarshal([]byte(vals[0]), &m)
 	if err != nil {
-		return nil, err
+		return nil, errInvalidQueryParams
```

Comment:
Wrap?

Reply:
Maybe it's not the right time to fix errors everywhere. I'll take it in the next PR.

Reply:
To fix, or not to fix?

Reply:
Not now, in the next PR.

Reply:
No, really, it's problematic, cause i don't wrap errors in the service and i don't check for error contains, so whole twins service would need to be refactored for errors, which i will do, but not now.

Reply:
Can we please immediately open an issue to track this?

Reply:
Issue opened. I'm resolvingt this conversation.